biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/bioenergy-source-class.ts

```diff
@@ -1,25 +1,48 @@
-import { Energy, EnergySource, Validatable, validate } from "@biogrid/grid-simulator";
+import {
+  Energy,
+  EnergySource,
+  Validatable,
+  validate,
+  ItemPosition,
+  Distance,
+} from '@biogrid/grid-simulator';
 
 export class BioEnergySource implements EnergySource {
   private sourceCapacity: Energy;
```

Comment:
@RolandNaijuka I am unsure here. Can you take a look?

Reply:
sourceCapacity has a setter and getter, and minCapacity has a getter (and can be considered as readonly) but sourceCapacity is not readonly since it is supposed to change

Reply:
when does sourceCapacity change?